recipes/antlr4-cppruntime/all/conanfile.py

```diff
@@ -23,7 +23,7 @@ class Antlr4CppRuntimeConan(ConanFile):
         "fPIC": True,
     }
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_find_package", "pkg_config"
```

Comment:
```suggestion
    generators = "cmake", "pkg_config"
```

`cmake_find_package` not required anymore